src/test/java/org/folio/rest/api/TestBaseWithInventoryUtil.java

```diff
@@ -118,6 +120,11 @@ public static void beforeAny() {
     LocationsTest.createLocation(fourthFloorLocationId,  FOURTH_FLOOR_LOCATION,  "TestBaseWI/FF");
   }
 
+  @Before
+  public void removeAllEvents() {
+    FakeKafkaConsumer.removeAllEvents();
```

Comment:
I agree it is a good idea to remove all of the captured messages before each test